integration/run_test.go

```diff
@@ -231,3 +233,83 @@ func TestFix(t *testing.T) {
 		t.Fatalf("testing error: %v", err)
 	}
 }
+
+func TestInit(t *testing.T) {
+	type testCase struct {
+		name string
+		dir  string
+		args []string
+	}
+
+	tests := []testCase{
+		{
+			name: "getting-started",
+			dir:  "../examples/getting-started",
+		},
+		{
+			name: "microservices",
+			dir:  "../examples/microservices",
+			args: []string{
+				"-a", "leeroy-app/Dockerfile=gcr.io/k8s-skaffold/leeroy-app",
+				"-a", "leeroy-web/Dockerfile=gcr.io/k8s-skaffold/leeroy-web",
+			},
+		},
+	}
+
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			oldYamlPath := filepath.Join(test.dir, "skaffold.yaml")
+			oldYaml, err := removeOldSkaffoldYaml(oldYamlPath)
```

Comment:
or you could just use `skaffold run -f skaffold.yaml.out`

Reply:
that's what I'm [doing](https://github.com/GoogleContainerTools/skaffold/pull/919/files/ff3c3b0d69e29100e0f118b21b6543f449d3efaf#diff-d858cc27041db377bd974728a3b9625aR285) :) the reason I had to do this funky business is because the example directories all have `skaffold.yaml`s in them, and `skaffold init` won't run if it detects a valid skaffold config in your directory. this just temporarily removes it so we can autogenerate one, then pass it to skaffold with `skaffold run -f <generatedYaml>`